client/components/marketing-survey/gsuite-cancel-purchase-dialog/index.jsx

```diff
@@ -235,12 +240,13 @@ GSuiteCancelPurchaseDialog.propTypes = {
 };
 
 export default connect(
-	( state, { purchase } ) => {
+	( state, { purchase, site } ) => {
 		return {
 			productName: getName( purchase ),
 			domain: purchase.meta,
 			purchasesError: getPurchasesError( state ),
 			userId: getCurrentUserId( state ),
+			domains: getDomainsBySiteId( state, site.ID ),
```

Comment:
Can we compute `selectedDomain` in the `connect()` code?

Also, the values aren't in alphabetical order. 😁 

```suggestion
		const siteDomains = getDomainsBySiteId( state, site.ID );
		return {
			domain: purchase.meta,
			productName: getName( purchase ),
			purchasesError: getPurchasesError( state ),
			selectedDomain: getSelectedDomain( { siteDomains, selectedDomainName: purchase.meta } ),
			userId: getCurrentUserId( state ),
```

Reply:
I hope I get used faster to the order of imports/properties 🤣